src/components/CardSingleSource.vue

```diff
@@ -0,0 +1,78 @@
+<script setup lang="ts">
+  /************************** 
+  Top level Card container
+  todo: fill out this section
+  ***************************/
+  
+  defineProps<{
+    card_title: string
+    card_description: string
+  }>()
+
+</script>
+
+<template>
+  <article class="card card-single-source">
+    
+    <header class="card-header">
+      <hgroup>
+        <h2 class="card-title">{{card_title}}</h2>
+        <span class="card-description">{{card_description}}</span>
+      </hgroup>
+    </header>
+
+    <section class="card-body">
+      <slot />
+    </section>
+
+  </article>
+</template>
+
+<style scoped lang="scss">
+
+  .card-single-source {
+
+    position: relative;
+  
+    .card-body {
+      padding: 0;
+    }
+
+    .card-header,
+    .card-footer {
+      position: relative;
+      z-index: 20;
+    }
+
+    &::before,
+    &::after {
+
+      content: '';
+
+      display: block;
+      width: $card-padding-internal;
+      height: 100%;
+
+      position: absolute;
+      z-index: 10;
+
+      background: linear-gradient(90deg, rgba(255,255,255,1) 0%, rgba(255,255,255,0) 100%);
```

Comment:
just a reminder for both of us to put any color into a var at some point

Reply:
Yup. As said above, that's coming out next branch. I intentionally made it hex so we'd both hate it and notice, lol. IT WORKED